fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -2062,17 +2134,6 @@ countries:
                     colormap: precip
                     lower_is_worse: yes
                     format: number4
-                ndvi-viirs:
-                    label: NDVI (VIIRS)
-                    path: djibouti/ndvi-viirs-mam.zarr
-                    var_names:
-                        value: NDVI
-                        lat: Y
-                        lon: X
-                        time: T
-                    colormap: precip
-                    lower_is_worse: yes
-                    format: number4
```

Comment:
Why did you delete this?

Reply:
Thanks! actually I don't know why I did it, I guess I got confused with the name and didn't realize it was Djibouti......fixed!